src/main/java/de/muenchen/kobit/backend/offer/repository/OfferRepository.java

```diff
@@ -0,0 +1,13 @@
+package de.muenchen.kobit.backend.offer.repository;
+
+import de.muenchen.kobit.backend.offer.model.Offer;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface OfferRepository extends JpaRepository<Offer, UUID> {
+
+    @Query("SELECT o FROM Offer o ORDER BY o.startDate desc")
+    List<Offer> findAllByOrderByStartDate();
```

Comment:
Die Query kannst du dir sparen. Kannst findAllByOrderBy nutzen https://stackoverflow.com/questions/25486583/how-to-use-orderby-with-findall-in-spring-data